src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! # Substrate
```

Comment:
Do we need a License here?

Reply:
yeah, I am going to use the license type used in the `sc-*` crates for now with version 0.0.0.